src/server/status/index.js

```diff
@@ -53,6 +56,9 @@ export default function (kbnServer, server, config) {
   server.route(wrapAuth({
     method: 'GET',
     path: '/status',
+    config: {
+      tags: ['api']
```

Comment:
I think the `api` tag on the `/api/status` route is probably fine - I can't think of any place in the UI where we request that.  However, the html `/status` page shouldn't have this tag, otherwise if a human tries to access the status page without being logged in, they are going to see a json 401 error in their browser rather than being redirected to the login page.